Classes/Command/CompressImagesCommand.php

```diff
@@ -95,8 +95,10 @@ protected function execute(InputInterface $input, OutputInterface $output): void
         foreach ($this->fileStorageRepository->findAll() as $fileStorage) {
             $excludeFolders = GeneralUtility::trimExplode(',', (string)$settings['excludeFolders'], true);
             $files = $this->fileRepository->findAllNonCompressedInStorageWithLimit($fileStorage, $limit, $excludeFolders);
-            $this->compressImages($files);
-            $this->clearPageCache();
+            if (count($files) > 0) {
```

Comment:
only use count() if there is always an array, othwise we'll get a PHP warning from PHP 7.2+. Safer it's to use `if (!empty($files))` plus it's also faster than count.